app-misc/khard/khard-0.17.0-r2.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
If possible, although it doesn't block this, try:
- EAPI 8
- [PEP517](https://projects.gentoo.org/python/guide/distutils.html#basic-use-pep-517-mode)

These are both nice-to-haves for what you're doing, but the next version bump should try to do it. So, you can treat this as a comment for @andrey-utkin instead if you wish.